website/_enemies/lab/golem-slime.md

```diff
@@ -11,3 +11,7 @@ hp: 9
 game_name: enemy_slime_beast
 sprite_folder_name: slimebeast
 ---
+
+Attacks with six small hitboxes travelling forward in the direction it's facing.
```

Comment:
Example of hitbox? Not necessary to address prior to merge, but would be nice to have.

Reply:
How do you think the example should be shown? would it be possible to embed an image in the entity description?

Reply:
Sure, use the path `website/assets/img/enemies` for images please, then embed using traditional Markdown syntax.

Reply:
You can do this in a separate PR since it should be somewhat trivial.